po/de.po

```diff
@@ -22746,13 +22745,13 @@ msgstr "Buchungsdatum fehlt."
 
 #: gnucash/price-quotes.scm:448
 msgid "No commodities marked for quote retrieval."
-msgstr "Keine Devisen/Wertpapiere zum Börsenkurs-Abruf markiert."
+msgstr "Keine Devisen/Wertpapiere zum Kurs-Abruf markiert."
```

Comment:
@CWehli 
Devisen: Fremdwährungen in Buchgeld;
Sorten: Fremdwährungen in Bargeld;.
Also besser 'Währungen oder'
und nochmal nach den beiden Begriffen durchsuchen und ersetzen.